### DISCUSSION TITLE
Error in Virtual User X: Error: missing close-bracket

### DISCUSSION THREAD

**Discussion by franciscofabion** [TPC-Council/HammerDB#639] [General]
Hi, I'm new to HammerDB, I'm creating a schema but I'm getting the error
Error in Virtual User 4: Error: missing close-bracket 
for all virtual users, could anyone give me a tip as to what could be causing this?

**Comment by sm-shaw** [comment]
Can you provide more details about what you are doing? e.g. GUI or CLI, database, operating system, options, etc. 
Also note that if there are special characters in a field such as a password you need to wrap the entry with curly brackets {} otherwise these will be interpreted.

**Reply by franciscofabion** [reply]
Thanks sm-shaw I am using GUIDE for database. there really were special characters in place of the {} keys
but now this error appears

Error in Virtual User 1: Error: [Microsoft][ODBC Driver 18 for SQL Server][SQL Server]CREATE DATABASE permission denied in database 'master'.
(executing the statement)

**Reply by sm-shaw** [reply]
OK great. We will think about doing something with the special characters, as it does catch people out. 
For the error `CREATE DATABASE permission denied in database 'master'.` you need to provide it a database name other than master such as tpcc. You will not have permission to write to the master database.
